### DISCUSSION TITLE
打棒球也能學數據分析？用AI培養孩子的數據素養

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#720] [General]
https://futurecity.cw.com.tw/article/2707
打棒球之餘算便學習東西，一舉兩得

**Comment by Valentinaa0214** [comment]
這種裝置雖然能即時回饋打擊者的數據與不足之處，並提升他們分析及判讀數據的素養，
但用電子設備體驗戶外的球類運動感覺還是少了一些實際打棒球的樂趣。

**Comment by HsuChen88** [comment]
就如同老師上課提到的AI教練，在各項運動中都可以導入AI來輔助練習。對於喜愛運動的我，這或許是一個未來可以努力的方向。

**Comment by yjpxi** [comment]
像這屆奧運，有的國家的桌球選手在練習的時候會用ai來幫你分析角度的問題，或是來分析自己的錯誤要怎麼改善，我覺得如果在選手中普及ai這個工具，那其實大家都可以比原本更強。
